### DISCUSSION TITLE
建議用戶下載支援Unihan的字型以免候選字變成後備字體（fallback font）

### DISCUSSION THREAD

**Discussion by 11201010** [forFudan/yuhao#46] [Show and tell]
目前有找到2種字體支援Unihan的字型：[BabelStone Han](https://babelstone.co.uk/Fonts/Han.html), [8suhU1](https://www.8suh.com/common-chinese-font-download/)。

對於windows用戶，在安裝這2種字體的ttf之後，打開RIME的用戶資料夾。
打開weasel.custom.yaml，輸入以下代碼：
```
patch:
  "style/font_face": "Yuniversus, BabelStone Han, 8suhU1"
```
如果想把星陳輸入法的候選字改為橫向排列，以星陳台灣簡碼為例，可以新增yustar_tw.custom.yaml，然後輸入以下代碼：
```
patch:
  "style/horizontal": true
```
存檔後，重新部署RIME即可。
![image](https://github.com/user-attachments/assets/d25c2972-a690-499a-98af-5e4dc26ed359)

**Comment by forfudan** [comment]
> 對於windows用戶，在安裝這2種字體的ttf之後，打開RIME的用戶資料夾。
> 
> 打開weasel.custom.yaml，輸入以下代碼：
> 
> ```
> 
> patch:
> 
>   "style/font_face": "Yuniversus, BabelStone Han, 8suhU1"
> 
> ```
> 

原來 windows可以選擇字體 fallback，我今天才知道。
